### PR TITLE
Test against Rails 5.1 and 5.2

### PR DESCRIPTION
**Pull request by lazyatom**
We are currently using Rails 5.1, and 5.2 will come soon.

### CODE REVIEW

**Review by andydust — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -7,5 +7,7 @@ gemfile:
   - gemfiles/rails_4.1.gemfile
```
Comment (federicokeen) [inline .travis.yml:7]: Guys how is this running in travis? We stopped paying for it.

**Review by federicokeen — APPROVED** [review]
Rest looks good to merge

File: .travis.yml (hunk 1 above)
Comment (andydust) [reply .travis.yml:7]: Open source projects are free on travis

Comment (federicokeen) [reply .travis.yml:7]: Ahh cool, just thought it be private by being under mubi

**Pull request merged**